lib/salesforce_ar_sync/version.rb

```diff
@@ -1,3 +1,3 @@
 module SalesforceArSync
-  VERSION = '5.1.0'
+  VERSION = '5.1.1'
```

Comment:
I feel like this isn't a patch/fix for anything and is giving a little extra functionality. 
Maybe bump the gem to 5.2.0 instead.